server/chainquery/models/ClaimModel.js

```diff
@@ -0,0 +1,177 @@
+const logger = require('winston');
+
+const {
+  assetDefaults: { thumbnail: defaultThumbnail },
+  details: { host }
+} = require('../../../config/siteConfig'); // TODO: Change back to '@config/siteConfig' when done testing
+
+const getterMethods = {
+  generated_extension() {
+    switch (this.content_type) {
+      case 'image/jpeg':
+      case 'image/jpg':
+        return 'jpg';
+      case 'image/png':
+        return 'png';
+      case 'image/gif':
+        return 'gif';
+      case 'video/mp4':
+        return 'mp4';
+      default:
+        logger.debug('setting unknown file type as file extension jpg');
+        return 'jpg';
+    }
+  },
+
+  // TODO: Factor this out.
+  generated_thumbnail() {
+    return this.thumbnail_url || defaultThumbnail;
+  },
+
+  generated_channel() {
+    console.log(this);
+    //
+  }
+}
+
+export default (sequelize, {
+  STRING, BOOLEAN, INTEGER, TEXT, DECIMAL
+}) => sequelize.define(
+  'claim',
+  {
+    id: {
+      primaryKey: true,
+      type: INTEGER,
+      set() { },
+    },
+    transaction_hash_id: {
+      type: STRING,
+      set() { },
+    },
+    vout: {
+      type: INTEGER,
+      set() { },
+    },
+    name: {
+      type: STRING,
+      set() { },
+    },
+    claim_id: {
+      type: STRING,
+      set() { },
+    },
+    claim_type: {
+      type: INTEGER,
+      set() { },
+    },
+    publisher_id: {
+      type: STRING,
+      set() { },
+    },
+    publisher_sig: {
+      type: STRING,
+      set() { },
+    },
+    certificate: {
+      type: STRING,
+      set() { },
+    },
+    sd_hash: {
+      type: STRING,
+      set() { },
+    },
+    transaction_time: {
+      type: INTEGER,
+      set() { },
+    },
+    version: {
+      type: STRING,
+      set() { },
+    },
+    value_as_hex: {
+      type: STRING,
+      set() { },
+    },
+    value_as_json: {
+      type: STRING,
+      set() { },
+    },
+    valid_at_height: {
+      type: INTEGER,
+      set() { },
+    },
+    height: {
+      type: INTEGER,
+      set() { },
+    },
+    effective_amount: {
+      type: INTEGER,
+      set() { },
+    },
+    author: {
+      type: STRING,
+      set() { },
+    },
+    description: {
+      type: STRING,
+      set() { },
+    },
+    content_type: {
+      type: STRING,
+      set() { },
+    },
+    is_nsfw: {
+      type: INTEGER,
+      set() { },
+    },
+    language: {
+      type: STRING,
+      set() { },
+    },
+    thumbnail_url: {
+      type: STRING,
+      set() { },
+    },
+    title: {
+      type: STRING,
+      set() { },
+    },
+    fee: {
+      type: DECIMAL(58, 8),
+      set() { },
+    },
+    fee_currency: {
+      type: STRING,
+      set() { },
+    },
+    is_filtered: {
+      type: INTEGER,
+      set() { },
+    },
+    bid_state: {
+      type: STRING,
```

Comment:
This is also an enumeration. I see that you set an enumeration above. Should this be listed as well? `Active,Expired,Controlling,Spent,Accepted` are the enumerations.